examples/end2end_tree_reconstruction_with_dstream_surf.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-set -euo pipefail
+set -eo pipefail
```

Comment:
Instead of disabling fail on unset, I'd mark the variables that might be unset with `${var:-}`